### DISCUSSION TITLE
Aw snap "STATUS_ACCESS_VIOLATION" on any WebGPU code in latest Chrome Canary?

### DISCUSSION THREAD

**Discussion by OmarShehata** [gpuweb/gpuweb#2407] [Q&A] [answered]
Just noticed this today, opening any WebGPU sample, like https://austin-eng.com/webgpu-samples/samples/helloTriangle, crashes immediately with:

```
Error code: STATUS_ACCESS_VIOLATION
```

I'm using Chrome Canary:

```
Version 99.0.4760.0 (Official Build) canary (64-bit)
```

Is anyone else able to reproduce? Could this be something changed in my setup? I don't see any recent issues for this on bugs.chromium.org (would this be the right place to report it?)

https://bugs.chromium.org/p/chromium/issues/list?q=STATUS_ACCESS_VIOLATION&can=2

**Comment by OmarShehata** [comment, marked as answer]
I re-installed chrome canary and the issue has gone away. It still appears to be the same version:

```
Version 99.0.4760.0 (Official Build) canary (64-bit)
```

**Reply by OmarShehata** [reply]
Now I'm encountering more strange behavior where `navigator.gpu` works fine on the WebGPU samples page but not in my localhost, which is very strange. I've switched to using Edge and that works for now.

**Reply by Kangz** [reply]
I think there's an issue in Chromium Canary at the moment in some A/B testing cohorts that triggers on `localhost` but we can't get reliable information on how to reproduce it :/ @austinEng I think I remember you linked to the issue somewhere last week?

Would you be willing to go in chrome://crashes, upload the corresponding crash and share the crash ID?

**Reply by OmarShehata** [reply]
Yes, I'm happy to do whatever I can to help you debug this! Here is one crash ID that is uploaded:

<html>
<body>
<h3 class="capture-time" style="margin: 0.75em 0px; font-size: 12px; color: rgb(0, 0, 0); font-family: &quot;Segoe UI&quot;, Tahoma, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Crash from Saturday, December 11, 2021 at 5:00:09 PM</h3>

Status: | Uploaded
-- | --
Uploaded Crash Report ID: | 448e24a220334613
Upload Time: | Saturday, December 11, 2021 at 5:00:10 PM


</body>
</html>

It is not crashing anymore since I reinstalled Canary, but the localhost issue remains.

**Reply by austinEng** [reply]
The crash is due to trials V8VirtualMemoryCage + PartitionAllocPCScanBlinkPartitions but has already been fixed. The localhost thing is probably separate. Sharing the Versions hashes in about://version might help us debug it.

Since this is a Chrome-specific issue, it might be best to file a bug report with the information at https://crbug.com/new instead of on the discussions tracker of the WebGPU standard.

**Reply by OmarShehata** [reply]
Thanks @austinEng! I opened https://bugs.chromium.org/p/chromium/issues/detail?id=1279617
